meta.go

```diff
@@ -95,3 +96,10 @@ func (m Meta) Check(log Logger, lvl Level, msg string) *CheckedMessage {
 	}
 	return NewCheckedMessage(log, lvl, msg)
 }
+
+// InternalError prints an internal error message to the configured
+// ErrorOutput
```

Comment:
Small nits, feel free to land once they're addressed:

1. Please add a period to the end of the sentence.
2. Mind adding an additional sentence explaining that this method is *only* for errors caused by bugs in the logger implementation? Any user-caused errors should just be logged to the application log.